dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -293,6 +300,49 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
           regional_partner_name: NO,
           committed: NO,
           able_to_attend_single: NO,
+          able_to_attend_multiple: NO,
+          principal_approval: YES, # Keep this as yes to test additional fields
+          schedule_confirmed: NO,
+          diversity_recruitment: NO,
+          free_lunch_percent: 0,
+          underrepresented_minority_percent: 0,
+          wont_replace_existing_course: nil,
+          csd_which_grades: NO,
+          csd_course_hours_per_year: NO,
+          previous_yearlong_cdo_pd: NO,
+          taught_in_past: 0
+        }, application.response_scores_hash
+      )
+    end
+
+    test 'autoscore for ambiguous responses to able_to_attend_multiple' do
+      application_hash = build :pd_teacher1819_application_hash, program: Pd::Application::Teacher1819Application::PROGRAM_OPTIONS.first
+      application_hash.merge!(
+        {
+          committed: Pd::Application::Teacher1819Application.options[:committed].last,
+          able_to_attend_multiple: "December 11-15, 2017 in Indiana, USA, #{Teacher1819ApplicationConstants::TEXT_FIELDS[:no_explain]}",
+          principal_approval: YES,
+          schedule_confirmed: NO,
+          diversity_recruitment: NO,
+          free_lunch_percent: '49.9%',
+          underrepresented_minority_percent: '49.9%',
+          wont_replace_existing_course: YES,
+          csd_which_grades: ['12'],
+          csd_course_hours_per_year: Pd::Application::ApplicationBase::COMMON_OPTIONS[:course_hours_per_year].last,
+          previous_yearlong_cdo_pd: ['Exploring Computer Science'],
+          taught_in_past: ['Exploring Computer Science']
```

Comment:
Are all these relevant to the test? Can we set it up more simply, just overriding `able_to_attend_multiple`?